### ISSUE TITLE
JasperReports Server Connection not working from Jaspersoft Studio Eclipse plugin

### ISSUE THREAD

**Issue by buddemat** [TIBCOSoftware/jaspersoft-studio-ce#28]
When I run Jaspersoft Studio as an Eclipse plugin, I cannot successfully create a JasperReports Server Connection from the Repository Explorer. Oddly, from Jaspersoft Studio as standalone version, there is no problem connecting to the exact same server. 

In Jaspersoft Studio, I open the "Create JasperReports Server Connection" dialog, enter my details (server & credentials) and click "Test connection". Everything works fine and it confirms "Test successful".  
In the Eclipse plugin, I open the dialog and enter my details exactly the same way. When I click "Test connection", I receive the error below.

I am running:

- Ubuntu 20.04.1 LTS 64 Bit
- TIBCO JasperReports Server, Product Version: 7.5.0, Build: 20191206_1440
- Eclipse IDE for Java Developers,  Version: 2019-12 (4.14.0), Build id: 20191212-1212 with Jaspersoft Studio feature 6.13.0.final

respectively

- Jaspersoft Studio 6.13.10 (with Jaspersoft Studio Designer 6.13.0.final as well)

Error message:

```
javax.ws.rs.InternalServerErrorException: HTTP 500 Internal Server Error
	at org.glassfish.jersey.message.internal.AbstractRootElementJaxbProvider.readFrom(AbstractRootElementJaxbProvider.java:126)
	at org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$TerminalReaderInterceptor.invokeReadFrom(ReaderInterceptorExecutor.java:264)
	at org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$TerminalReaderInterceptor.aroundReadFrom(ReaderInterceptorExecutor.java:234)
	at org.glassfish.jersey.message.internal.ReaderInterceptorExecutor.proceed(ReaderInterceptorExecutor.java:154)
	at org.glassfish.jersey.message.internal.MessageBodyFactory.readFrom(MessageBodyFactory.java:1124)
	at org.glassfish.jersey.message.internal.InboundMessageContext.readEntity(InboundMessageContext.java:851)
	at org.glassfish.jersey.message.internal.InboundMessageContext.readEntity(InboundMessageContext.java:783)
	at org.glassfish.jersey.client.ClientResponse.readEntity(ClientResponse.java:326)
	at org.glassfish.jersey.client.InboundJaxrsResponse$1.call(InboundJaxrsResponse.java:111)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:315)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:297)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:228)
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:399)
	at org.glassfish.jersey.client.InboundJaxrsResponse.readEntity(InboundJaxrsResponse.java:108)
	at com.jaspersoft.studio.server.protocol.restv2.ARestV2ConnectionJersey.toObj(ARestV2ConnectionJersey.java:45)
	at com.jaspersoft.studio.server.protocol.restv2.RestV2ConnectionJersey.getServerInfo(RestV2ConnectionJersey.java:332)
	at com.jaspersoft.studio.server.protocol.restv2.RestV2ConnectionJersey.connect(RestV2ConnectionJersey.java:224)
	at com.jaspersoft.studio.server.protocol.ProxyConnection.connect(ProxyConnection.java:117)
	at com.jaspersoft.studio.server.WSClientHelper.checkConnection(WSClientHelper.java:91)
	at com.jaspersoft.studio.server.wizard.ServerProfileWizard.connect(ServerProfileWizard.java:94)
	at com.jaspersoft.studio.server.wizard.ServerProfileWizard.access$1(ServerProfileWizard.java:89)
	at com.jaspersoft.studio.server.wizard.ServerProfileWizard$2.run(ServerProfileWizard.java:68)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:122)
Caused by: javax.xml.bind.JAXBException
 - with linked exception:
[java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory]
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:241)
	at javax.xml.bind.ContextFinder.find(ContextFinder.java:455)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:652)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:599)
	at org.glassfish.jersey.message.internal.AbstractJaxbProvider.getStoredJaxbContext(AbstractJaxbProvider.java:258)
	at org.glassfish.jersey.message.internal.AbstractJaxbProvider.getJAXBContext(AbstractJaxbProvider.java:250)
	at org.glassfish.jersey.message.internal.AbstractJaxbProvider.getUnmarshaller(AbstractJaxbProvider.java:178)
	at org.glassfish.jersey.message.internal.AbstractJaxbProvider.getUnmarshaller(AbstractJaxbProvider.java:154)
	at org.glassfish.jersey.message.internal.AbstractRootElementJaxbProvider.readFrom(AbstractRootElementJaxbProvider.java:122)
	... 22 more
Caused by: java.lang.ClassNotFoundException: com.sun.xml.internal.bind.v2.ContextFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at org.eclipse.osgi.internal.framework.ContextFinder.loadClass(ContextFinder.java:145)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at javax.xml.bind.ContextFinder.safeLoadClass(ContextFinder.java:573)
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:239)
	... 30 more
```

**Comment by saysource** [comment]
Hi @buddemat , do you see any stack trace in the server log? The error would suggest an error on the server side.

**Comment by gt78** [comment]
Hi @buddemat , do you see any error in the server log? The stack trace suggests an error on the server-side (HTTP 500), but the ClassNotFoundException seems pretty off as consequence.

**Comment by gt78** [comment]
Another possible reason is that you are using java 11 in your Eclipse platform. They may be a problem since the class com.sun.xml.internal.bind.v2.ContextFactory has been removed from JDK 8.

**Comment by buddemat** [comment]
Hi @gt78, thanks for the quicl reply. You're correct, the java version seems to be the root of the problem: ```java.runtime.version=11.0.8+10-post-Ubuntu-0ubuntu120.04```

**Comment by maxrabbi** [comment]
The problems with the Java 11 environment should have been fixed/improved with the latest 6.15.0 recently released.
